stylelint.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
```

Comment:
@skjnldsv I guess we should also move this to a `@nextcloud` package.

Reply:
Yes, there is none yet :shrug: